Krimirundgang/StopActivity.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+
+namespace Krimirundgang
+{
+    [Activity(Label = "StopActivity")]
+    public class StopActivity : Activity
+    {
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+            
+            // Set our view from the "main" layout resource
+            SetContentView(Resource.Layout.Stop);
+
+            int stopID = Intent.GetIntExtra("StopId", 1);
+            Stop stop = new Tour().Stops.First(s => s.StopID == stopID);
+
```

Comment:
We should investigate whether it's viable to try and pass the stop as a whole instead of its Id. That would save us some annoyances